website/docs/developer/api/api.md

```diff
@@ -10,13 +10,16 @@ The purpose of this page is to give developers information on how to use the ORC
 All ORCA APIs use the `POST` method.
 All API endpoints use AWS IAM authorization.
 
+When using the Load Balancer endpoint for GraphQL, encode your query as a JSON string, and store it in the request body.
```

Comment:
might be helpful to show an example of how to do this in request body.

Reply:
On second thought, you're right. Even if limited, it's good to have a concrete example. Added.